bemani/utils/read.py

```diff
@@ -1344,6 +1344,27 @@ def scrape(self, xmlfile: str) -> Tuple[List[Dict[str, Any]], List[Dict[str, Any
             songid = int(music_entry.find('music_id').text)
             bpm_min = float(music_entry.find('bpm_min').text)
             bpm_max = float(music_entry.find('bpm_max').text)
+            earliest_version = 0
+            version = int(music_entry.find('version').text.strip(), 16)
```

Comment:
Interesting, so this is a bitfield, and its available on all supported game xmls? Wonder how I missed this.

Reply:
Wow, so it is....this is a good find! I think its fine to say "hey if you want fully correct versions, re-import all the XML files."